### PR TITLE
🎨 [Frontend] Sort folders alphabetically

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Merry xmas @wvangeit!

SortedFolders

## Related issue/s

- closes https://github.com/ITISFoundation/osparc-simcore/issues/6788

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/static-webserver/client/source/class/osparc/dashboard/WorkspacesAndFoldersTree.js [hunk 1]
```diff
@@ -228,6 +228,9 @@ qx.Class.define("osparc.dashboard.WorkspacesAndFoldersTree", {
 
       const sharedWorkspaceModel = this.__getModel(-1, null);
       sharedWorkspaceModel.getChildren().append(workspaceModel);
+      sharedWorkspaceModel.getChildren().sort(((a, b) => {
```
Comment (wvangeit) [inline services/static-webserver/client/source/class/osparc/dashboard/WorkspacesAndFoldersTree.js:231]: Just a small question, I assume this is case-aware. I wonder if it maybe shouldn't be?

**Comment by wvangeit** [comment]
Thx 🎄 ⛄ 🎄 ⛄ 🎄 ⛄

**Review by matusdrobuliak66 — COMMENTED** [review]
Q: I would assume, as a user, that the default sorting should be by "last modified"?

**Review by GitHK — APPROVED** [review]

**Comment by odeimaiz** [comment]
> Q: I would assume, as a user, that the default sorting should be by "last modified"?

In the tree(s) alphabetically sorted (always), in the main view, depends on your selection

**Comment by wvangeit** [comment]
> > Q: I would assume, as a user, that the default sorting should be by "last modified"?
> 
> In the tree(s) alphabetically sorted (always), in the main view, depends on your selection

Which also seems to be consistent with how e.g. gdrive allows one to move a folder.

File: services/static-webserver/client/source/class/osparc/dashboard/WorkspacesAndFoldersTree.js (hunk 1 above)
Comment (jsaq007) [reply services/static-webserver/client/source/class/osparc/dashboard/WorkspacesAndFoldersTree.js:231]: agreed. Don't we have a utils function for sorting?

**Review by jsaq007 — COMMENTED** [review]
Personally I agree with @wvangeit. Otherwise approved

File: services/static-webserver/client/source/class/osparc/dashboard/WorkspacesAndFoldersTree.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/dashboard/WorkspacesAndFoldersTree.js:231]: By default it isn't
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator#sensitivity

**Review by sanderegg — COMMENTED** [review]
:+1:

**Comment by odeimaiz** [comment]
> Personally I agree with @wvangeit. Otherwise approved

What do you want me to implement so that I get your approval, sir?

**Review by wvangeit — APPROVED** [review]

**Review by pcrespov — COMMENTED** [review]
:+1:

**Pull request merged**